edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html

```diff
@@ -107,29 +75,8 @@ <h3>
 </div>
 
 
-    <button type="submit" class="action action-primary action-update js-login login-button"><%- gettext("Sign in") %></button>
-
```

Comment:
Why was this block removed? @luismorenolopera

Reply:
move to login.underscore